doc/user/content/sql/system-catalog/mz_internal.md

```diff
@@ -213,6 +213,21 @@ of each process in each cluster replica in the system.
 | `reason`     | [`text`]                     | If the cluster replica is in a `offline` state, the reason (if available). For example, `oom-killed`.   |
 | `updated_at` | [`timestamp with time zone`] | The time at which the status was last updated.                                                          |
 
+## `mz_cluster_replica_status_history`
+
+{{< warn-if-unreleased v0.116 >}}
+The `mz_cluster_replica_status_history` table records status changes
+for all processes of all extant cluster replicas.
+
+<!-- RELATION_SPEC mz_internal.mz_cluster_replica_status_history -->
+| Field         | Type      | Meaning
+|---------------|-----------|--------
+| `replica_id`  | [`text`]  | The ID of a cluster replica.
+| `process_id`  | [`uint8`] | An identifier of a process within the replica.
```

Comment:
Just wondering if that should be "The identifier" (or maybe "The ID") instead of "An identifier"

```suggestion
| `process_id`  | [`uint8`] | The identifier of a process within the replica.
```

Reply:
Fine by me! FWIW, the `mz_internal` docs are currently pretty inconsistent with the column descriptions, both variants are used. But it makes sense to at least keep things consistent within the same table.